codespell_lib/data/dictionary.txt

```diff
@@ -7630,7 +7633,7 @@ deamiguate->disambiguate
 deamiguates->disambiguates
 deamiguation->disambiguation
 deamon->daemon
-deamonified->daemonified
+deamonified->daemonized
```

Comment:
Suggest the -sed version too, although this feels like a made-up computer word.

Reply:
i made it (more) consistend.

Reply:
> consistend

Do you want to add that typo too? :laughing:

Reply:
since i'm not alone with that mistake: #1646